site.yml.sample

```diff
@@ -76,6 +76,8 @@
         name: ceph-defaults
     - import_role:
         name: ceph-validate
+    - import_role:
```

Comment:
Nit, usually we prefer to call `ceph-facts` right after `ceph-defaults`. Thanks.

Reply:
OK, let me reorder that :)